### PR TITLE
新增主题 halo-theme-fresh

### PR DESCRIPTION
**Pull request by adozhao**
新增主题 halo-theme-fresh

```release-note
None
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@
 - [theme-mainroad](https://github.com/liuchangfitcloud/theme-mainroad) - 移植于 Hugo 主题 [Vimux Mainroad](https://github.com/Vimux/Mainroad)，适用于 Halo 2.0 的 Mainroad 主题
 - [halo-theme-meaword](https://github.com/Meayair/Halo-Theme-MeaWord) - 适用于 Halo 2.0 的 MeaWord 主题
 - [halo-theme-zero](https://github.com/adozhao/halo-theme-zero) - 适用于 Halo 2.0 的 Zero 主题
+- [halo-theme-zero](https://github.com/adozhao/halo-theme-fresh) - 一款清新的 Halo 2.0 主题
```
Comment (JohnNiang) [inline README.md:46]:
```suggestion
- [halo-theme-fresh](https://github.com/adozhao/halo-theme-fresh) - 一款清新的 Halo 2.0 主题
```

**Review by JohnNiang — CHANGES_REQUESTED** [review]

**Review by JohnNiang — APPROVED** [review]
Thanks for your contribution!

/approve

**Review by ruibaby — APPROVED** [review]
/lgtm

**Comment by ruibaby** [comment]
https://www.halo.run/store/apps/app-owNyx

**Pull request merged**